CONTRIBUTING.md

```diff
@@ -63,21 +63,43 @@ When reporting an issue, details are key. Include the following:
+* **tracked** - this issue is in the JIRA backlog for the team working on snap
+* **reviewed/duplicate** - used to tag issues which are identical to other issues _OR_ which are resolved by the same fix of another issue (either case)
+* **reviewed/wont-fix** - the universal sign that we won't fix this issue. This tag is important to use as we separate out the nice-to-have features from the maintainer's agreed upon strategic direction
+* **wip-do-not-merge** - specifically for a PR to indicate it is not ready for review just yet
+
+The difference between bugs, features and enhancements can be confusing. To be extra clear, we reduced it down to two options. Here are their definitions: 
+* **type/bug** - the classic definition of missing or misbehaving code from existing functionality (this includes malfunctioning tests)
+* **type/feature-or-enhancement** - any new functionality or improvements/enhancements to existing functionality. We use one label because it's prioritized in identical ways during sprint planning
+
+For the sake of clarity, here are a few scenarios you might see play out. 
+
+As a maintainer: 
+* An issue is opened stating that Snap is not working. Upon review, the maintainer finds it is an issue with a plugin. She will label the issue with **type/wrong-repo** and open a new issue under the plugin where she tags the original issue reporter, links the original issue and labels it **bug** (which is available in plugins repositories).
+* An issue is opened stating that Snap is not working. It turns out to be related to a Snap's functionality. The maintainer will label it **type/bug**. She takes on the bug, labels in **reviewed/work-in-progress**. She finds it maps to the Scheduler package and adds additional context with **component/scheduler**. She opens a PR while she fixes this issue. In the PR body, it says "Fixes #" for the previous issue and labels it **wip-do-not-merge**.
+
+And as a contributor:
+* A contributor has an idea to improve Snap. He opens an issue with guidelines on how to fix it. A maintainer likes the idea, label it **type/feature-or-enhancement**. Once a maintainer or contributor begins working on the issue, it's labeled **reviewed/in-process**. A pull request is opened and labeled **wip-work-in-progress**.
+* A contributor has an idea to improve Snap. He opens an issue with guidelines on how to fix it and the maintainer labels it **type/feature-or-enhancement**. A maintainer believes the approach requires more user input and labels it **type/rfc** to indicate it's an open discussion on implementation. Once a maintainer or contributor begins working on the issue, it's labeled **reviewed/in-process**. A pull request is opened and labeled **wip-work-in-progress**. Whoever authors the PR should check back on the original issues thread for further feedback until code is merged.
```

Comment:
As above, **reviewed/in-process** should be replaced by **reviewed/in-progress**